buildbot/configure.py

```diff
@@ -247,6 +250,7 @@ def main():
     parser.add_argument("--ci-defaults", action="store_true", help="Enable default CI parameters")
     parser.add_argument("--enable-plugin", action='append', help="Enable SYCL plugin")
     parser.add_argument("--disable-fusion", action="store_true", help="Disable the kernel fusion JIT compiler")
+    parser.add_argument("--add_opt_set", type=str, choices=['none', 'default', 'sanitize'], default='none', help="Enables recommended extra options for compile & link. Two values are supported: 'default' and 'sanitize'. 'Sanitize' option is an extension of 'default' set.")
```

Comment:
Would `default='none'` result in branch at line 185 always being taken? Can we make it `default=None` instead?